flask/main.py

```diff
@@ -71,7 +71,6 @@ def checkout():
     print("> /checkout inventory", inventory)
 
     with sentry_sdk.start_span(op="process_order", description="function"):
-        wait(operator.ge, 14, .5)
```

Comment:
This 'wait was causing transactons+errors not to be linked to each other (Distributed Tracing), so removed it.

Reply:
@cstavitsky